portability-types-common/src/main/java/org/datatransferproject/types/common/models/photos/PhotosContainerResource.java

```diff
@@ -161,4 +161,27 @@ void ensureRootAlbum(boolean allowRootPhotos){
       this.albums = albums_;
     }
   }
+
+  // Replaces forbidden characters and makes sure that the name is not too long
+  void ensureCleanAlbumNames(String forbiddenTitleCharacters, char replacementCharacter, int maxTitleLength) {
```

Comment:
Can you add tests for these new behaviors: https://github.com/google/data-transfer-project/blob/master/portability-types-common/src/test/java/org/datatransferproject/types/common/models/photos/PhotosContainerResourceTest.java